pox-locking/src/events.rs

```diff
@@ -210,19 +240,28 @@ fn create_event_info_data_code(
                         max-amount: {max_amount},
                         ;; equal to args[4]
                         auth-id: {auth_id},
+                        ;; Get end cycle ID
+                        end-cycle-id: (some (burn-height-to-reward-cycle unlock-height)),
+                        ;; Get start cycle ID
+                        start-cycle-id: (+ (burn-height-to-reward-cycle unlock-height) prepare-offset),
```

Comment:
I think this is wrong - this is saying the start cycle is the cycle in which the stacker unlocks

Reply:
Agree +1
I updated these -- imo the prepare_offset can always only come from `burn_block_height`, and in some cases is not needed at all.